### ISSUE TITLE
isDateTime and isDuration return undefined instead of false

### ISSUE THREAD

**Issue by vin-car** [moment/luxon#376]
Both `isDateTime` and `isDuration` return `undefined` when parameter is not a valid `DateTime`/`Duration`.

The documentation states that the return type is `boolean`, but the result is `true` or the _falsy_ `undefined`, both functions never give `false` as output.

Step to reproduce (v1.8.2), simply execute:

```js
console.log( DateTime.isDateTime({}),  Duration.isDuration({}) );
```

---

Why not implementing

```js
static isDateTime(o) {
    return o instanceof DateTime;
}
```

and 

```js
static isDuration(o) {
    return o.isLuxonDuration;
}
```

over `return o.isLuxonDateTime;` and `return o.isLuxonDuration;`?

**Comment by icambron** [comment]
We can't use `instanceof` because it's not reliable in cross-context environments. Instead, to fix this, let's just add `|| false`

**Comment by icambron** [comment]
Fixed by #383
